### ISSUE TITLE
Markdown images are not being exported

### ISSUE THREAD

**Issue by rgonzga** [ipython/ipython#8341]
I put up the server with the following command:

ipython notebook --no-browser --ip="*"

Then, when trying to export as PDF, HTML or LaTeX the images inserted with the <img> tag in Markdown cells are not exported.

**Comment by koenvb** [comment]
* Could you explain which version of python and iptyhon you are using so I could try to reproduce?
* Could you provide an example notebook I could try to see whats wrong?

**Comment by rgonzga** [comment]
Hello

The python version is 2.7.6, while the ipython version is 3.0.0. Also I
have installed  pandoc 1.13.2. I'm attaching an example notebook and the
PDF generated with the File>Download as...>PDF option. The PNG is shown in
the markdown cell in of the notebook but not in the PDF. Also the generated
LaTeX doesn't have the image.

Thank you

2015-04-23 23:49 GMT-05:00 koenvb <notifications@github.com>:

>
>    - Could you explain which version of python and iptyhon you are using
>    so I could try to reproduce?
>    - Could you provide an example notebook I could try to see whats wrong?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ipython/ipython/issues/8341#issuecomment-95799823>.
>



-- 
Rafael González García

**Comment by DucQuang1** [comment]
I also the same problem when exporting (download as) pdf option on ipython. 
My packages versions are: python version 2.7.9, ipython 3.0.0, I installed pandoc 1.13.2 
I also installed MikTex version 5633 to generate the Latex documents.

**Comment by takluyver** [comment]
Are the images in question local or on a remote URL? It might be useful to see the .tex produced by nbconvert (exporting to PDF actually exports to latex and then runs latex to build the PDF).

Unfortunately, the attachments before didn't work - Github issues don't have attachments. You can put .ipynb and .tex files on https://gist.github.com/ and link them from here. That won't work for PDFs, but I think they're less helpful for debugging the problem anyway.

**Comment by jakobgager** [comment]
Pandoc strips raw html code when converting to latex (see [docs](http://pandoc.org/README.html#raw-html)), hence ``<img>`` won't work here. However these tags should survive a conversion to html.

**Comment by rgonzga** [comment]
Yes, I discovered that the <img> tag is part of the problem. But when 
substituting it with the markdown command, i.e. ![image](picture.png), 
the pdf exporting also fails. By reading the output of the terminal I 
suppose that the problem is that the server or the LaTeX system (pdftex 
in this case) expects to have the image files in its working directory. 
So, my workaround was to export as markdown and use pandoc to convert 
the markdown file to the final pdf. Maybe there's another way to do it 
but I find it better to have a folder containing all the figures next to 
the notebook file rather than having them scattered around in the same 
folder.

El 05/06/15 a las 15:23, Jakob Gager escribió:
>
> Pandoc strips raw html code when converting to latex (see docs 
> <http://pandoc.org/README.html#raw-html>), hence |<img>| won't work 
> here. However these tags should survive a conversion to html.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/ipython/ipython/issues/8341#issuecomment-109429604>.
>

-- 
Rafael González García

**Comment by hagger98** [comment]
Please see the following code of auto generated Latex file created by Jupyter Notebook (ipynb) file. You can open the .tex extension file in the editor and go to the line 32 or 33 where you will find the bold line and comment it out. Your problem will be partially solved. 
**Code:** 
 \makeatletter
    \def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth
    \else\Gin@nat@width\fi}
    \makeatother
    \let\Oldincludegraphics\includegraphics
    % Set max figure width to be 80% of text width, for now hardcoded.
    **#\renewcommand{\includegraphics}[1]{\Oldincludegraphics[width=.8\maxwidth]{#1}}**
After commenting it out, find the image attachment command in .tex file. When you find that lines where compiler stops usually. **replace the line of:**
**\adjustmentbox 
with: 
 \includegraphics[scale=0.5 or whatever you desire]{yourimage.png}**
